pkgs/top-level/perl-packages.nix

```diff
@@ -12122,6 +12122,25 @@ let self = _self // overrides; _self = with self; {
     };
   };
 
+  PerconaToolkit = buildPerlPackage rec {
+    name = "Percona-Toolkit-3.0.11";
+    src = fetchFromGitHub {
+      owner = "percona";
+      repo = "percona-toolkit";
+      rev = "${version}";
+      sha256 = "18wxvp7psqrx0zdvg47azrals572hv9fx1s3p0q65s87lnk3q63l";
+    };
+    outputs = [ "out" ];
```

Comment:
`out` is the default output. You don't have to specify it here.

Reply:
Error build without utputs = [ "out" ]; - 
nix build -f ~/works/src-nix/nixpkgs-my perlPackages.PerconaToolkit                                                     
builder for '/nix/store/6vv63471a14kdpprsrpa51g5gm1kwdhd-perl-Percona-Toolkit-3.0.11.drv' failed to produce output path '/nix/store/yx40qkihv0rm3psccm5d53j2pi0mxf19-perl-Percona-Toolkit-3.0.11-devdoc'
[0 built (1 failed), 0.0 MiB DL]
error: build of '/nix/store/6vv63471a14kdpprsrpa51g5gm1kwdhd-perl-Percona-Toolkit-3.0.11.drv' failed